NitroxClient/MonoBehaviours/MovementController.cs

```diff
@@ -0,0 +1,171 @@
+using NitroxClient.Communication.Abstract;
```

Comment:
This class needs refactor. Too many code smells/not using our coding conventions to mark each issue individually....

Reply:
I made some changes, not sure if that is all.